### ISSUE TITLE
Create packet for different distro (DEB, RPM, PKGBUILD)

### ISSUE THREAD

**Issue by MauroMombelli** [MauroMombelli/opengpg-skt#1]
Commit 50befaa can b considered complete enought to be realeased on the wild

**Comment by bam80** [comment]
Also, it needs Wireless Tools version 30 (beta), latest stable version 29 doesn't fit.
BTW, latest stable release was over 10 years ago, and the tools is currently considered [deprecated](https://en.wikipedia.org/wiki/Wireless_tools_for_Linux#Alternatives), so it seems doubtful design choice. Was there a reason for that?
Anyway, I couldn't built it since my distro provides only stable version of the Wireless Tools :(

**Comment by MauroMombelli** [comment]
my understanding is Wireless Tool is "iw", and it still developed by the kernel mantainer: https://git.kernel.org/pub/scm/linux/kernel/git/jberg/iw.git

That part of the code is signed for rewrite anyway, so if you have suggestion for better library please let me know!

**Comment by bam80** [comment]
>my understanding is Wireless Tool is "iw"

No, these are different utils. Excerpt from Wikipedia [article](https://en.wikipedia.org/wiki/Wireless_tools_for_Linux#Alternatives):
>The wireless tools package is currently considered deprecated; the alternative being the more recent iw utility.

This project depends on old Wireless Tools currently, so rewriting it for iw seems would be OK

**Comment by MauroMombelli** [comment]
i'll look into this, opefully i will have some time this weekend

2018-04-23 18:27 GMT+02:00 bam80 <notifications@github.com>:

**Comment by bam80** [comment]
>if your distro doesn't ship something this old

My distro just ships something that stable :)
Ok, I see your point. Not sure if  `libiw` deprecated also or not, it wasn't my statement. Maybe you are right.

**Comment by bam80** [comment]
Can it be distributed in distro-agnostic way, e.g. Flatpak?

**Comment by MauroMombelli** [comment]
Yes i guess it could but i think after 3 years there should be no problem
